<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QMainWindow" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1443</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>大恒相机推流监控软件</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="imagePixmap">
      <property name="minimumSize">
       <size>
        <width>1024</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(135, 135, 135);</string>
      </property>
      <property name="text">
       <string>未获取任何图像</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="menuDock">
   <property name="styleSheet">
    <string notr="true">* {
font: 12pt;
}

QPushButton:!enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.653409 rgba(183, 187, 186, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(165, 165, 165);
	font: 12pt ;
	border:none
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.653409 rgba(51, 141, 177, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(165, 165, 165);
	font: 12pt ;
	border:none
}

QPushButton{
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.653409 rgba(113, 158, 177, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(255, 255, 255);
	font: 12pt;
}
</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>属性配置</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="menuLayout">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <item row="6" column="1">
      <widget class="QCheckBox" name="opencvBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Opencv独立窗口预览</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QPushButton" name="folderButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="grabButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>采集</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="freezeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>停止</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(89, 121, 170);
color: rgb(255, 255, 255);
font: 14pt;</string>
       </property>
       <property name="text">
        <string>图像处理信息</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="scanButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>重启程序</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="deviceCountEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(230, 230, 230);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>流服务器地址：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="showHudBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>图片上方显示HUD信息</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>设备名称：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="pushBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>推流至流媒体服务器</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="2">
      <widget class="QLineEdit" name="serverEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="deviceNameEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(230, 230, 230);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>保存路径：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="intervalSpin">
       <property name="toolTip">
        <string>输入范围100-100,000</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>设备数量：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>保存帧间隔：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(89, 121, 170);
color: rgb(255, 255, 255);
font: 14pt;</string>
       </property>
       <property name="text">
        <string>连接信息</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="saveBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>每隔一定帧数保存图像</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="savePathEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="connectButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>连接相机</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="logDock">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 246, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="logLayout">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QTextEdit" name="logText">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="daheng_stream_pusher.qrc"/>
 </resources>
 <connections/>
</ui>
